core/util/constants.py

```diff
@@ -1,10 +1,13 @@
-RAW_DATA_PATH="leopidotera-dk/multimedia.txt"
-RAW_LABEL_PATH="leopidotera-dk/occurrence.txt"
-DATASET_PATH="leopidotera-dk.csv"
-IMGDIR_PATH = "image_db/"
-LABEL_DATASET_PATH = "occurrence.csv"
-MERGE_COLS = ['genericName', 'species', 'family', 'stateProvince', 'gbifID', 'identifier', 'format', 'created',
-              'iucnRedListCategory', 'lifeStage', 'sex']
-BFLY_FAMILY = ['Pieridae', 'Papilionidae', 'Lycaenidae', 'Riodinidae', 'Nymphalidae', 'Hesperiidae', 'Hedylidae']
-BFLY_LIFESTAGE = ['Pupa', 'Caterpillar', 'Larva']
-FEATURE_DIR_PATH = "image"
\ No newline at end of file
+RAW_DATA_PATH="leopidotera-dk/multimedia.txt"
+RAW_LABEL_PATH="leopidotera-dk/occurrence.txt"
+DATASET_PATH="leopidotera-dk.csv"
+IMGDIR_PATH = "image_db/"
+LABEL_DATASET_PATH = "occurrence.csv"
+MERGE_COLS = ['genericName', 'species', 'family', 'stateProvince', 'gbifID', 'identifier', 'format', 'created',
+              'iucnRedListCategory', 'lifeStage', 'sex']
+BFLY_FAMILY = ['Pieridae', 'Papilionidae', 'Lycaenidae', 'Riodinidae', 'Nymphalidae', 'Hesperiidae', 'Hedylidae']
+BFLY_LIFESTAGE = ['Pupa', 'Caterpillar', 'Larva']
+FEATURE_DIR_PATH = "image"
+WINDOWS_COMPLIANT_OS_SEP = "/"

```

Comment:
```suggestion
PATH_SEP = "/"
```
It's the same on osx & linux.